### ISSUE TITLE
How to run a test set?

### ISSUE THREAD

**Issue by kinghuin** [magic282/SEASS#1]
If we use model.eval() to make a test, it needs dict['tgt'] which just like a cheat. So how to make a test? Thanks a lot

**Comment by kinghuin** [comment]
just run translate.py
